xmipp

```diff
@@ -277,10 +277,13 @@ def pull():
 def cloneOrCheckout(repo, branch):
     repo_dir = os.path.join('src', repo) if repo != XMIPP else '.'
     if branch not in getAllBranchesAndTags(repo)[1]:
```

Comment:
`getAllBranchesAndTags(repo)[1]` is redundant.
`getAllBranchesAndTags(repo)` returns True, branch when it works, and, when it does not, it returns nothing because the program exits. Therefore, True is not necessary, because if things don't work inside `getAllBranchesAndTags()`, there will never be a return.
Additionally, why bother running a command that gets all branches and tags and looks for the specfiic one you need in that list, while you can run a git command that looks for a specific branch. If it returns True, branch exists, else clone HEAD.